app/utils/ScheduledTask.scala

```diff
@@ -0,0 +1,50 @@
+package utils
+
+import akka.agent.Agent
+import play.api.Logger
+import play.api.libs.concurrent.Akka
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+trait ScheduledTask[T] {
```

Comment:
This class should probably be removed - `ScheduledTask` isn't used anywhere in this pull request, as far as I can see?